packages/tsx-editor/src/transpiler/transpile.types.tsx

```diff
@@ -0,0 +1,9 @@
+export interface ITranspiledOutput {
```

Comment:
Most of the time for things besides React components, as long as there are only a small number of interfaces, we put them in the same file as the code. But either way is probably fine.

Reply:
I decided to just add the interface in the same file :)